tests/dragonfly/acl_family_test.py

```diff
@@ -189,3 +189,36 @@ async def test_acl_categories_multi_exec_squash(df_local_factory):
 
     await admin_client.close()
     await client.close()
+
+
+@pytest.mark.asyncio
+async def test_acl_deluser(df_local_factory):
+    df = df_local_factory.create(multi_exec_squash=True, port=1111)
```

Comment:
It has actually the same issue I mentioned in the previous pr 😢

Reply:
Are you referring that I am explicitly creating the dfly instance? We need two clients, currently with `async_client` there can only be one (since I can't pass the same fixture twice, eg, (async_client, async_client)

Reply:
There is the df_server fixture which is the currently default running instance, you can get its port via df_server.port